tools/perf/test.js

```diff
@@ -14,7 +14,7 @@ program
   .usage('[options] [command]')
   .option('-e, --engine <string>', 'Engine to test: node, jsshell, d8 [node]',
           'node')
-  .option('-s, --sample <int>', 'Sample size [30]', 30)
+  .option('-s, --sampleSize <int>', 'Sample size [30]', 30)
```

Comment:
Why change this? If needed, please use `--sample-size`.

Reply:
to separate sample name from sample size.

Reply:
reverted and renamed sampleName to benchmarkName